Lib/test/_test_multiprocessing.py

```diff
@@ -5730,7 +5730,10 @@ def check_resource_tracker_death(self, signum, should_die):
         pid = _resource_tracker._pid
 
         os.kill(pid, signum)
-        time.sleep(1.0)  # give it time to die
+        if sys.platform.startswith("netbsd") and should_die:
+            time.sleep(60.0 * 3)  # give it more time to die on netbsd
```

Comment:
do we know _why_ it takes so long?

Reply:
Unfortunately, a 30-second timeout is too short, and the test fails. I thought about using `support.LONG_TIMEOUT`, but which is over 3 minutes.

Reply:
thanks for changing it to a skip!